SQL/CRUD/hw2.sql

```diff
@@ -0,0 +1,22 @@
+/* 
+<problem>
+Write a update query to modified value for 'commission' is 'commission'-.02
+for agents who have minimum payment amount of 'customer' table
+ */
+
+use sql_test_asaka;
+
+SELECT * FROM agents;
+
+UPDATE agents
+SET COMMISSION = COMMISSION - .02
+WHERE AGENT_CODE = (
+    SELECT AGENT_CODE
+    FROM customer
+    WHERE PAYMENT_AMT = (
+        SELECT MIN(PAYMENT_AMT)
+        FROM customer
+    )
```

Comment:
I think you misread the problem. 
> gents who have minimum payment amount of 'customer' table

An agent has many customers, so you have to find the agents have the total amount from their customer is minimum.